doc/source/changes.rst

```diff
@@ -9,12 +9,133 @@ users familiar with older versions of the documentation who want
 only a quick update to the docs without reading the entire set
 of documentation again from scratch.
 
+This list is NOT a comprehensive list of everything.  Specifically,
+minor one-line changes, or bug fixes, are not mentioned here.
+
+Most importantly, changes that might have broken previously working
+scripts are not always signposted here.  To be sure, you should read
+the change log in the main github repository, which is repeated in the
+release announcements that are made in various places with each
+release.
+
 .. contents::
     :local:
     :depth: 3
 
 ****
 
+Changes in 0.19.0
+-----------------
+
+Art asset changes
+:::::::::::::::::
+
+Though not represented in these documents, numerous changes to the
+part models and artwork are included as part of this update, including
+the new KAL9000 high-end computer part.
+
+Varying Power Consumption
+:::::::::::::::::::::::::
+
+:ref:`Electrical drain <electricdrain>` is now handled in a dynamically
+changing way that actually notices how much you are using the CPU and
+uses less power if the CPU is mostly idling (if it spends most of its
+time on WAIT statements).
+
+For mods that want to re-balance the meaning of electric charge units,
+the drain factor is also editable in
+:ref:`module config fields <kospartmodule>` in the various ``part.cfg``
+files the mod ships with.  This opens them up to being changed by
+ModuleManager rules.
+
+Delegates (function pointers)
+:::::::::::::::::::::::::::::
+
+User functions and built-in functions (but not suffixes yet) can
+now be referred to with function pointers called :ref:`delegates <delegates>`
+along with "currying" of pre-loaded arguments.
+
+Optional Defaulted Parameters
+:::::::::::::::::::::::::::::
+
+:ref:`User functions <user_functions>` and user programs can now be
```

Comment:
That's not a sphinx tag which is why I didn't.  The only way to do it that way is to drop out to HTML URL which I'm trying to avoid, or add a ``.. _blarg:`` rest target to the destination location, which I supposed I could do.